### PR TITLE
Bc data

### PR DESCRIPTION
**Pull request by jenreiher-bcgov**
Add overview summary of bcdata content

### CODE REVIEW

File: docs/working-with-data.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Working with Data
+
+The BC Data Service (formerly known as Data BC) encourages and enables the strategic management and sharing of data across the government enterprise and with the public.
```
Comment (mdunhamwilkie) [inline docs/working-with-data.md:3]: As far as I know, DataBC is still a service on its own, offered by the BC Data Service Division along with other programs like DIP (Data Innovation Program). Even within our branch (DSS) there are teams working on non-DataBC stuff - e.g., my team (Data Equity Services) and the API team.

File: docs/working-with-data.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### API Services Portal
+
+If you are a developer or part of a government team, you can administer and manage your APIs using the [API Services Portal](https://api.gov.bc.ca/).
+
+## Other Resources
+
+### BC Data Catalogue
+
+[The BC Data Catalogue](https://www2.gov.bc.ca/gov/content/data/bc-data-catalogue) provides the easiest access to government's data holdings, as well as applications and web services. Thousands of the datasets discoverable in the Catalogue are available under the Open Government License - British Columbia.
```
Comment (mdunhamwilkie) [inline docs/working-with-data.md:29]: Perhaps add a link for Open Government Licence - British Columbia (https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc).  Also note spelling of "Licence" - nonUS version.

Also, I'd change "as well as applications and web services" to "as well as their associated applications and web services". The intent is to describe data holdings as the primary entries, and apps and web services as resources attached to those entries.

**Review by mdunhamwilkie — CHANGES_REQUESTED** [review]
Just added a couple of comments inline.  You may also want to run this page by the SPM's of the various DataBC teams - Terry Lanktree (Data Publication Services), Laurissa Barnes-Roberts (Mapping Services), Alix Cote (Location Services) and Sunil Mysore (Data Discovery and Distribution Services)

File: docs/working-with-data.md (hunk 1 above)
Comment (jenreiher-bcgov) [reply docs/working-with-data.md:3]: aha good to know! My intel was false. I will remove this. thanks!

**Comment by jenreiher-bcgov** [comment]
@Shivanganii - got this input back from the folks on the data side they are keen to get this merged in
are you able to do 90% feedback on this, and if needed add it to your card sort exercise?

File: docs/working-with-data.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+[The BC Data Catalogue](https://www2.gov.bc.ca/gov/content/data/bc-data-catalogue) provides the easiest access to government's data holdings, as well as their associated applications and web services. Thousands of the datasets discoverable in the Catalogue are available under the [Open Government License - British Columbia](https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc).
+
+### Geospatial Data
+
+The province's area, varied topography and predominantly natural resource based economy has positioned B.C. as a leader in geographic information management and generates a wealth of geographic information that is discoverable in the BC Data Catalogue.  BC Data Service provides access to this information through a suite of unique applications and services that comprise [the B.C. Spatial Data Infrastructure](https://www2.gov.bc.ca/gov/content/data/geographic-data-services).
+
+## Standards
+
+* [API Guidelines](https://classic.developer.gov.bc.ca/Data-and-APIs/BC-Government-API-Guidelines)
```
Comment (Shivanganii) [inline docs/working-with-data.md:37]: Can we move this further up, at the end of 'Work with Data' section? I feel standards should always be "front and centre".

**Comment by Shivanganii** [comment]
> @Shivanganii - got this input back from the folks on the data side they are keen to get this merged in are you able to do 90% feedback on this, and if needed add it to your card sort exercise?

Sounds good, only 1 comment from me and I'll add this section as a card.

**Pull request merged**